frontend/src/pages/FeatureConfigurationPage.tsx

```diff
@@ -119,7 +119,9 @@ const FeatureConfigurationPage = () => {
                         onClick={() => navigate(paths.features)}
                         disabled={false}
                     >
-                        <FormattedMessage id={"features.config.cancel"}></FormattedMessage>
+                        <FormattedMessage id={isValid
+                          ? "features.config.cancel"
+                          : "features.config.back"}></FormattedMessage>
```

Comment:
I've added this because "Annuleren" looked weird when you just saved the changes

Reply:
Good one, I was thinking about this last night hahah, nice yo see you noticed it as well